From 872e183c0f7dbf85e310d9f09e93d624cdfa1572 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Wed, 8 Nov 2023 09:37:35 +0100
Subject: [PATCH 382/841] fix(stm32mp2): increase BL31 size

Increase BL31 size to 116kB to avoid compilation issues.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 045a0d35fc87595148c90b4605b088bc46dd2ee5 fix(stm32mp2): increase BL31 size]
Change-Id: I74164562369899e68ac849af1d440cba7369e977
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/338357
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 plat/st/stm32mp2/stm32mp2_def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/st/stm32mp2/stm32mp2_def.h b/plat/st/stm32mp2/stm32mp2_def.h
index 695b9144c..40604815a 100644
--- a/plat/st/stm32mp2/stm32mp2_def.h
+++ b/plat/st/stm32mp2/stm32mp2_def.h
@@ -140,7 +140,7 @@ enum ddr_type {
 						 (STM32MP_PARAM_LOAD_SIZE +	\
 						  STM32MP_HEADER_SIZE))
 
-#define STM32MP_BL31_SIZE		U(0x0001C000)	/* 112 KB for BL31 */
+#define STM32MP_BL31_SIZE		U(0x0001D000)	/* 116 KB for BL31 */
 
 #define STM32MP_BL2_RO_SIZE		U(0x00018000)	/* 96 KB */
 
-- 
2.47.2

